include/proxsuite/proxqp/dense/solver.hpp

```diff
@@ -985,6 +985,12 @@ qp_solve( //
   T primal_feasibility_in_lhs(0);
   T dual_feasibility_lhs(0);
 
+  T duality_gap(0);
+  T rhs_duality_gap(0);
+  isize max_dim = std::max(qpmodel.dim,qpmodel.n_eq);
+  max_dim = std::max(max_dim,qpmodel.n_in);
```

Comment:
```suggestion
  const isize max_dim = std::max(qpmodel.dim,std::max(qpmodel.n_eq,qpmodel.n_in));
```

Reply:
OK (it is integrated in the commit below)